docs/docs/kb/09-docker.md

```diff
@@ -23,7 +23,7 @@ After you scaffold and start a chain in your Docker container, all Ignite CLI co
 
 ```bash
 docker run -ti ignitehq/cli -h
-docker run -ti ignitehq/cli scaffold chain github.com/test/planet
+docker run -ti ignitehq/cli scaffold chain planet
 docker run -ti ignitehq/cli chain serve
```

Comment:
Are we still using `ignitehq/cli` and not `ignite/cli`?

Reply:
From what @lubtd was telling me yesterday, `ignite` was not available for us to use on docker, so we're still using `ignitehq`. But @lubtd please provide any additional details.

Reply:
Yes, `ignitehq` is our handle on Docker